gui/src/components/dashboard/cortx-dashboard-info-card.vue

```diff
@@ -0,0 +1,95 @@
+/* * CORTX-CSM: CORTX Management web and CLI interface. * Copyright (c) 2020
+Seagate Technology LLC and/or its Affiliates * This program is free software:
+you can redistribute it and/or modify * it under the terms of the GNU Affero
+General Public License as published * by the Free Software Foundation, either
+version 3 of the License, or * (at your option) any later version. * This
+program is distributed in the hope that it will be useful, * but WITHOUT ANY
+WARRANTY; without even the implied warranty of * MERCHANTABILITY or FITNESS FOR
+A PARTICULAR PURPOSE. See the * GNU Affero General Public License for more
+details. * You should have received a copy of the GNU Affero General Public
+License * along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing, * please email
+opensource@seagate.com or cortx-questions@seagate.com. */
+<template>
+  <div class="info-card-container" @click="callBack(navPath)">
+    <div class="dashboard-card-icon" :class="iconClass"></div>
+    <div class="card-info">
+      <span class="title">{{ title }}</span>
+      <span class="description">{{ description }}</span>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from "vue-property-decorator";
+
+@Component({
+  name: "cortx-dashboard-info-card",
+  components: {}
+})
+export default class CortxDashboardInfoCard extends Vue {
+  @Prop({ required: true }) public iconClass: string;
+  @Prop({ required: true }) public title: string;
+  @Prop({ required: true }) public description: string;
+  @Prop({ required: true }) public navPath: string;
+  @Prop({ required: true }) public callBack: Function;
+}
+</script>
+
+<style lang="scss" scoped>
+.info-card-container {
+  padding: 0.75em;
+  border: 1px solid #e5e5e5;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 4px;
+  cursor: pointer;
+  box-shadow: none;
+  transition: box-shadow 200ms ease-in-out;
+}
+.info-card-container:hover {
+  box-shadow: 0px 6px 15px #e5e5e5;
+}
+.card-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 1em;
+}
+.dashboard-card-icon {
+  height: 60px;
+  width: 60px;
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+// Below classes are for alert icons
```

Comment:
all background Image should come from their parent component since this is a common component  used for both health card and alert.

Reply:
In many places, they have followed this method and I have just followed the convention.

Reply:
This will create issue if you try to change the card i.e. adding or removing particular card, using same component for other widget or section on dashboard. you will need to add image urls for all the places you use it for.

Reply:
I have updated this component take imageUrl as a prop. Please review and provide your feedback.